packages/react-instantsearch-core/src/connectors/connectHitInsights.ts

```diff
@@ -33,6 +33,12 @@ function inferPayload({
   const { index } = results;
   const queryID = currentHit.__queryID;
   const objectIDs = [currentHit.objectID];
+
+  if (!queryID) {
+    throw new Error(`Could not infer \`queryID\`. Ensure \`clickAnalytics: true\` was added with the Configure widget.
+See: https://alg.li/lNiZZ7`);
```

Comment:
Here we could be pointing to the [React doc](https://www.algolia.com/doc/guides/getting-insights-and-analytics/search-analytics/click-through-and-conversions/how-to/send-click-and-conversion-events-with-instantsearch/react/#prepare-react-instantsearch-to-connect-with-insights-client-for-javascript) directly.

Reply:
Good point! changed it to https://alg.li/VpPpLt